pkg/serverless/invocationlifecycle/constants.go

```diff
@@ -18,5 +18,7 @@ const (
 	// used in /lambda/end-invocation
 	InvocationErrorHeader = "x-datadog-invocation-error"
 
-	parentIDHeader = "x-datadog-parent-id"
+	// ParentIDHeader is the header containing the parentID
+	// used in  /trace.go
+	ParentIDHeader = "x-datadog-parent-id"
```

Comment:
We also need to account for the sampling priority header here: `x-datadog-sampling-priority`